db/subscribers.go

```diff
@@ -0,0 +1,128 @@
+package db
```

Comment:
helper functions, still to work within the database, this time to help managing subscribers to channels